src/main/java/net/dries007/tfc/util/OreDictionaryHelper.java

```diff
@@ -121,6 +121,9 @@ public static void init()
         OreDictionary.registerOre("alabasterRaw", new ItemStack(BlocksTFC.ALABASTER_RAW_PLAIN));
         OreDictionary.registerOre("alabasterPolished", new ItemStack(BlocksTFC.ALABASTER_POLISHED_PLAIN));
 
+        //barrel oredict for use in powderkeg recipe
+        BlocksTFC.getAllBarrelItemBlocks().forEach(barrel -> OreDictionary.registerOre("barrel", new ItemStack(barrel)));
+
```

Comment:
Don't add this here, for our items they're all generally done in the constructor. i.e.
`OreDictionaryHelper.register(this, "barrel")`

Reply:
Interesting. Now I see the variants are registered here, and the single items register in constructors. Thanks.